### ISSUE TITLE
Stop ingesting `project_urls` from the napari hub configuration file

### ISSUE THREAD

**Issue by neuromusic** [chanzuckerberg/napari-hub#633]
This is a task defined in https://github.com/chanzuckerberg/napari-hub/issues/629, see that parent ticket for more context.

**Comment by richaagarwal** [comment]
@neuromusic I'm going to move this out of "Ready" since it seems that the other tasks defined in #629 need to happen first - let me know if that's not the case!

**Comment by richaagarwal** [comment]
This will be superseded by the work that @DragaDoncila is doing (captured in https://docs.google.com/document/d/1XNf0KOpJqj1Tzyu9u6KZRdkBTP7m6xSRLExJ6KPhMfM/edit#heading=h.zi22evmfyjbu), so I am closing it here.
